docs/Integrations/payments/revenuecat.md

```diff
@@ -0,0 +1,238 @@
+---
+slug: revenuecat
+title: RevenueCat
+tags: []
+description: Learn how to integrate RevenueCat in your FlutterFlow app.
+sidebar_position: 1
+---
+
+# RevenueCat
+
+[RevenueCat](https://www.revenuecat.com/) simplifies implementing in-app purchases and subscriptions by handling all purchase validation operations.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'}}>
+    <iframe 
+        src="https://demo.arcade.software/z3Fw57EiyR05PY6OPMG6?embed&show_copy_link=true"
+        title=""
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+<p></p>
+
+Integrating the RevenueCat in your app comprises the following steps:
+
+1. [Setup RevenueCat](#1-setup-revenuecat)
+2. [Enable RevenueCat in FlutterFlow](#2-enable-revenuecat-in-flutterflow)
+3. [Retrieving in-app purchases and subscription details](#3-retrieving-in-app-purchases-and-subscription-details)
+4. [Add RevenueCat actions](#4-add-revenuecat-actions)
+5. [Testing](#5-testing)
+6. [Launch](#6-launch)
+
+## 1. Setup RevenueCat
+
+To set up the RevenueCat, follow these steps carefully:
+
+1. Sign up for a new RevenueCat account [here](https://app.revenuecat.com/).
```

Comment:
Add a space after each bullet points so they are not tied together.

Reply:
I think we should do this at CSS level by adding something like this 
`ol li{
  padding: 3px;
}`

WDYT?